content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -14,34 +14,29 @@ Role-based access control (RBAC) is a method of regulating access to computer or
 
 {{% capture body %}}
 
-`RBAC` uses the `rbac.authorization.k8s.io` {{< glossary_tooltip text="API group" term_id="api-group" >}}
-to drive authorization decisions, allowing you to dynamically configure policies
-through the Kubernetes API.
+RBAC authorization uses the `rbac.authorization.k8s.io`
+{{< glossary_tooltip text="API group" term_id="api-group" >}} to drive authorization
+decisions, allowing you to dynamically configure policies through the Kubernetes API.
 
 To enable RBAC, start the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
 with `--authorization-mode=RBAC` (this is the default).
 
-## API Overview
+## API objects
 
-The RBAC API declares four top-level types which will be covered in this
-section. Users can interact with these resources as they would with any other
-API resource (via `kubectl`, API calls, etc.). For instance,
-`kubectl apply -f (resource).yml` can be used with any of these examples,
-though readers who wish to follow along should review the section on
-bootstrapping first.
+The RBAC API declares four kinds of Kubernetes object: _Role_, _ClusterRole_,
+_RoleBinding_ and _ClusterRoleBinding_.
 
 ### Role and ClusterRole
 
-In the RBAC API, a role contains rules that represent a set of permissions.
+An RBAC _Role_ or _ClusterRole_ contains rules that represent a set of permissions.
 Permissions are purely additive (there are no "deny" rules).
-A role can be defined within a namespace with a `Role`, or cluster-wide with a `ClusterRole`.
```

Comment:
this statement is helpful to frame the difference between Role and ClusterRole objects

Reply:
I've added:
```diff
diff --git a/content/en/docs/reference/access-authn-authz/rbac.md b/content/en/docs/reference/access-authn-authz/rbac.md
index 74ee497f0..91e16b968 100644
--- a/content/en/docs/reference/access-authn-authz/rbac.md
+++ b/content/en/docs/reference/access-authn-authz/rbac.md
@@ -55,6 +55,9 @@ ClusterRoles have several uses. You can use a ClusterRole to:
 1. define permissions on namespaced resources and be granted across all namespaces
 1. define permissions on cluster-scoped resources
 
+If you want to define a role within a namespace, use a Role; if you want to define
+a role cluster-wide, use a ClusterRole.
+
 #### Role example
 
 Here's an example Role in the "default" namespace that can be used to grant read access to
```

Reply:
I found that these statements were worded differently than I expected (or I required an intro or further glue):
`If you want to define a role within a namespace, use a Role; if you want to define ...`
You are providing recommendations or which resource to use?

Also read the content in the list above and got hung up on `be granted access`
define permissions on namespaced resources and grant access across [over] all namespaces?